app/views/dispatch/_messages.html.haml

```diff
@@ -1,12 +1,13 @@
-.messages
-  - @conversation.messages.order('created_at asc').each do |message|
-    .message{style: "text-align: #{(message.sent_by == 'Voter') ? 'left' : 'right'}"}
-      %code
-        %span.msg-epoch
-          = message.created_at.to_i
-        , #{message.sent_by}
-      .msg-body
-        = message.body
+- if @conversation.user.phone_number != '5555555555'
```

Comment:
Should we use 'phone_number_normalized' instead, in case it ever ends up in there as 555-555-5555, or some other variant?